### PR TITLE
PERF - Monitoring Windows EC2 Instances using CloudWatch

### PR DESCRIPTION
**Pull request by ericpullen**
There are 2 parts to this PR:

1. I created a new Lab that will guide a customer through the process of deploying a Windows EC2 instance and monitoring its CPU performance and Memory pressure.
1. I have added 2 new classes to the CSS that will add a small 2px black border to any image and gives it a small corner radius. 
lab_picture_auto - Just adds the black border and radius 
lab_picture_small - This also reduces the width to 75%

You can use those features by adding ?classes=lab_picture_auto or ?classes=lab_picture_small to the end of any picture in Markdown. 
Example:
`CreateDashBoard1`

### CODE REVIEW

File: layouts/shortcodes/common/CreateEC2KeyPair.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!--
```
Comment (setheliot) [inline layouts/shortcodes/common/CreateEC2KeyPair.md:1]: Do you use this?

https://github.com/awslabs/aws-well-architected-labs/pull/174#issuecomment-732509472

File: layouts/shortcodes/common/CreateEC2KeyPair.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+     keypairname: What you want the user to name the keypair
```
Comment (setheliot) [inline layouts/shortcodes/common/CreateEC2KeyPair.md:2]: Please add a parameter to specify which type of KeyPair to create. Also adding region would be helpful: 

https://github.com/awslabs/aws-well-architected-labs/pull/174/files#

**Review by setheliot — CHANGES_REQUESTED** [review]

File: content/Performance Efficiency/100_Labs/100_Monitoring_Windows_EC2_CloudWatch/5_generating_load.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+Write-Warning "To cancel execution of all jobs, close the PowerShell Host Window (or terminate the remote session)"
+Write-Output "=-=-=-=-=-=-=-=-=-=  CPUs in box: $cpus =-=-=-=-=-=-=-=-=-= "
+
+
+
+# This will stress the CPU
+foreach ($loopnumber in 1..$cpus){
+  Start-Job -ScriptBlock{
+  $result = 1
+      foreach ($number in 1..2147483647){
```
Comment (setheliot) [inline content/Performance Efficiency/100_Labs/100_Monitoring_Windows_EC2_CloudWatch/5_generating_load.md:56]: nitpick- you changed it in the actual code, but forgot to change it here.

2147483647 --> 0x7FFFFFFF

**Review by setheliot — APPROVED** [review]

File: content/Performance Efficiency/100_Labs/100_Monitoring_Windows_EC2_CloudWatch/5_generating_load.md (hunk 3 above)
Comment (ericpullen) [reply content/Performance Efficiency/100_Labs/100_Monitoring_Windows_EC2_CloudWatch/5_generating_load.md:56]: Oops, fixed it in master.

**Pull request merged**